docs/index.js

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable no-alert */
 /* eslint-disable no-plusplus */
 /* eslint-disable no-unused-vars */
+
```

Comment:
- [ ] Please make sure you don't disable the linter and fix them manually. Linter helps you to write clean code.